prime-router/docs/azure_database_login.md

```diff
@@ -0,0 +1,46 @@
+# Azure Database Login
+
+This is the process for logging into our database via Azure Active Directory.
+
+## Login to Azure CLI
+
+```aidl
+az login
+```
+
+## Request a token for Active Directory
+
+```aidl
+az account get-access-token --resource-type oss-rdbms
+```
+
```

Comment:
I got an error for the `oss-rdbms` resource type...
```
rickhawes@MacBook-Pro-16 ~/P/p/prime-router (master)> az account get-access-token --resource-type oss-rdbmsk
az account get-access-token: 'oss-rdbmsk' is not a valid value for '--resource-type'.
Did you mean 'oss-rdbms' ?

TRY THIS:
az account get-access-token
Get an access token for the current account

https://docs.microsoft.com/en-US/cli/azure/account#az_account_get_access_token
Read more about the command in reference docs
```

Reply:
@RickHawesUSDS From your output, looks like a typo in the command.

Reply:
Hmm. There was.